.github/workflows/docs.yml

```diff
@@ -54,6 +54,7 @@ jobs:
             && pip3 install .
             && pip3 install openfermionpyscf
             && pip3 install aiohttp fsspec h5py
+            && pip3 install -e git+https://github.com/quantumlib/Qualtran.git
```

Comment:
Is this really needed for the docs? What is the plan for updating this and using? Also, why `-e` is needed?

Reply:
I can probably delete the -e directive, but I don't think the docs will build without installing qualtran's main branch. We will update this to `pip3 install qualtran` when Qualtran has their new release to v1.0

Reply:
I deleted this since it works without this installation.